HOW_TO_DOCS.md

```diff
@@ -12,10 +12,23 @@ Most important things to know:
 
   + For now, the content is all flat.   All 'md' files are at this top level directory.   There are no subdirectories
   + The main sidebar is _data/sidebars/home_sidebar.yml.   If you create a new page, find an appropriate header under which to place it.  As of now, this is the only sidebar being used
+  + All images must include a width and height in order to be valid on the AMP version of the documentation
   + Local references to files in this directory take the following form:  
  ```
  
 [Create a PX-Enterprise Cluster](create-px-enterprise-cluster.html)  <br/>
  
  ```
 Note the reference to a **".html"** file, which gets generated automatically by jekyll from the **".md"** file.
+
+
+## AMP Version
+
+There is an AMP version of the documentation which is effectively a drop in replacement for the `page.html` layout. 
+When you wish to generate the AMP documentation, you must replace the `page.html` file in `_layouts/` with `amp.html` - this process may change in the future.
```

Comment:
Not sure how this fits in with the below.  Sounds like we need to do something, but then it seems we don't.